_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -103,27 +110,63 @@ Heading marked up with `role="heading"` that describes the topic or purpose of i
 
 #### Passed example 6
 
-Heading marked up with `h` element that describes the topic or purpose of its section of the content. The heading is visible, but is not included in the Accessibility Tree.
+The heading marked up with `h` element describes the topic or purpose of its section of the content. The heading is visible, but is not included in the Accessibility Tree.
+
+```html
+<h1 aria-hidden="true">Opening Hours</h1>
+<p>We are open Monday through Friday from 10 to 16</p>
+```
+
+#### Passed example 7
+
+The heading marked up with `h` element describes the topic or purpose of a part of its section of the content.
+
+```html
+<h1>Oranges</h1>
+<p>I really like oranges.</p>
+<p>Apples are great too, though.</p>
+```
+
+#### Passed example 8
+
+The heading marked up with `h` element doesn't have a section of content.
+
+```html
+<h1>Oranges on sale</h1>
+```
+
+#### Passed example 9
+
+The heading marked up with `h` element has visible text nodes and an accessible name that both describe the content, though the two are different.
 
 ```html
-<h1 class="target" aria-hidden="true">Opening Hours</h1>
+<h1 aria-label="Office opening hours">Opening Hours</h1>
 <p>We are open Monday through Friday from 10 to 16</p>
 ```
 
+#### Passed example 10
+
+The visible heading is made up of an image of text, that also has an accessible name. Both the visible content and the accessible name is describe the content.
```

Comment:
Grammar ... "Both the visible content and the accessible name describe the content." 

Remove "is"